Identity/AzureStack.Identity.Common.psm1

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# See LICENSE.txt in the project root for license information.
+
+<#
+.Synopsis
+    Initialize the Azure RM environment
+#>
+function Initialize-AzureRmEnvironment
+{
+    [CmdletBinding()]
+    param
+    (
+        # The endpoint of the Azure Stack Resource Manager service.
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNull()]
+        [ValidateScript( {$_.Scheme -eq [System.Uri]::UriSchemeHttps})]
+        [uri] $AdminResourceManagerEndpoint,
```

Comment:
nit: if this is going to be common, this should just say "ResourceManagerEndpoint" as it can apply to either public or admin ARM